src/Schema/Directives/BaseDirective.php

```diff
@@ -41,6 +43,13 @@ abstract class BaseDirective implements Directive
      */
     protected $definitionNode;
 
+    /**
+     * Cached directive arguments.
+     *
+     * @var array<string, mixed>|null
+     */
+    protected $directiveArgs = null;
```

Comment:
```suggestion
     * Lazily initialized.
     *
     * @var array<string, mixed>
     */
    protected $directiveArgs;
```

We don't want `null` assigned to this property, it is just uninitialized at first.